aixplain/factories/script_factory.py

```diff
@@ -0,0 +1,31 @@
+import os
+import json
+from typing import Tuple
+
+import requests
+
+from aixplain.utils import config
+
+
+class ScriptFactory:
+
+    @classmethod
+    def upload_script(cls, script_path: str) -> Tuple[str, str]:
```

Comment:
For every factory, please create a corresponding `Asset` into aixplain.modules

Reply:
I only followed the current convention to upload a script using backend. What kind of asset we need here or i did not understand.  I referred to `FileFactory` while putting this and that factory does not use any kind of Asset. Do you mean sth else?